lib/src/models/multireddit.dart

```diff
@@ -3,12 +3,162 @@
+    // Direct Link: (https://www.reddit.com/dev/api/#DELETE_api_multi_{multipath}).
+    await reddit.put(url, body: data);
+    // TODO(ckartik): Research if we should GET subreddits.
+  }
+
+  /// Copy this [Multireddit], and return the new [Multireddit].
+  ///
+  /// [displayName] is an optional string that will become the display name of the new
+  /// multireddit and be used as the source for the [name]. If [displayName] is not
+  /// provided, the [name] and [displayName] of the muti being copied will be used.
+  ///
+  /// Returns  an instance of a [Multireddit].
+  Future<Multireddit> copy([String displayName]) async {
+    final String url = apiPath['multireddit_copy'];
+    final String name = sluggify(displayName) ?? _name;
+
+    displayName ??= _displayName;
+
+    final Map data = {
+      kDisplayName: displayName,
+      kFrom: _path,
+      kTo: apiPath['multiredit']
+          .replaceAll(_multiredditRegExp, name)
+          .replaceAll(User.userRegExp, reddit.user.me()),
+    };
+    return await reddit.post(url, data);
+  }
+
+  /// Delete this [multireddit].
+  Future delete() async {
+    await reddit.delete(_infoPath);
+  }
+
+  /// Remove a [Subreddit] from this [Multireddit].
+  ///
+  /// [subreddit] is a string containing the name of the subreddit to be deleted.
+  Future remove(String subreddit) async {
+    final String url = apiPath['multireddit_update']
+        .replaceAll(_multiredditRegExp, _name)
+        .replaceAll(User.userRegExp, _author)
+        .replaceAll(Subreddit.subredditRegExp, subreddit);
+    final Map data = {'model': "{'name': $subreddit}"};
+    await reddit.delete(url, body: data);
+  }
+
+  /// Rename this [Multireddit].
+  ///
+  /// [displayName] is the new display for this [multireddit].
+  /// The [name] will be auto generated from the displayName.
+  Future rename(displayName) async {
+    final String url = apiPath['multireddit_rename'];
+    final Map data = {
+      kFrom: _path,
+      kDisplayName: _displayName,
+    };
+    await reddit.post(url, data);
+    _displayName = displayName;
+  }
+
+  /// Update this [Multireddit].
+  ///
+  /// [newSettings] is a map potentially containing a list of keyword args
+  /// that should be updated. These include:
+  /// [display_name]: the display_name for the multireddit.
+  /// [subreddits]: A list of subreddits in this multireddit.
+  /// [description_md]: A description of this multireddit, formated in markdown.
+  /// [icon_name]: can be one of: [art and design], [ask], [books], [business],
+  /// [cars], [comic], [cute animals], [diy], [entertainment], [food and drink],
+  /// [funny], [games], [grooming], [health], [life advice], [military],
+  /// [models pinup], [music], [news], [philosophy], [pictures and gifs], [science],
+  /// [shopping], [sports], [style], [tech], [travel], [unusual stories], [video],
+  /// or [None].
+  /// [key_color]: RGB Hex color code of the form i.e "#FFFFFF".
```

Comment:
Let's pull in the [color](https://pub.dartlang.org/packages/color) package for this and let the user pass in a `Color` object. That way, we don't have to restrict them to hex color codes and there should be code that handles out of range values.